fixtures/additional-modules/wrangler.toml

```diff
@@ -0,0 +1,9 @@
+name = "additional-modules"
+main = "src/index.ts"
+compatibility_date = "2023-08-01"
+
+find_additional_modules = true
+rules = [
+  { type = "CommonJS", globs = ["**/*.cjs"]},
```

Comment:
It doesn't look like the CJS rule is being tested. Could you add a CJS module?

Reply:
Adding